.github/workflows/packages.yml

```diff
@@ -59,8 +59,8 @@ jobs:
         include:
           - {tag: manylinux2014, arch: x86_64}
           - {tag: manylinux2014, arch: i686}
-          - {tag: manylinux_2_24, arch: aarch64}
-          - {tag: manylinux_2_24, arch: ppc64le}
+          - {tag: manylinux_2_28, arch: aarch64}
+          - {tag: manylinux_2_28, arch: ppc64le}
```

Comment:
`PSYCOPG2_TESTDB_HOST` may need to be set to a hostname instead of an IP in CI. Received this error during run:

```sh
[root@e558eccedeff psycopg2]# PLAT=manylinux_2_28_aarch64 PACKAGE_NAME=psycopg2-binary PSYCOPG2_TESTDB=postgres PSYCOPG2_TESTDB_HOST=172.172.17.0.3 PSYCOPG2_TESTDB_USER=postgres PSYCOPG2_TESTDB_PASSWORD=password PSYCOPG2_TEST_FAST=1 ./scripts/build/build_manylinux_2_28.sh
#...
+ /opt/python/cp310-cp310/bin/python -c 'import tests; tests.unittest.main(defaultTest='\''tests.test_suite'\'')'
Failed connection to test db: OperationalError could not translate host name "172.172.17.0.3" to address: Name or service not known

Please set env vars 'PSYCOPG2_TESTDB*' to valid values.
```

Reply:
Huh? there's a 172 too much there, no?

Reply:
I think newer versions of the PostgreSQL client connection library assume/insist on a hostname instead of an IP. I’ve seen similar errors using `psql` as well.

Reply:
`172.172.17.0.3` is not a valid IP. Where is it coming from?

Reply:
Since I can't run the GitHub Actions workflow locally I'm having to manually spin up a Postgres container to run the tests against (as you can see above).

The error is not that it can't connect (ping works) but that Postgres client fails to connect to the defined name (for some reason it interpreted `172.172.17.0.3` as a hostname and not an IP) 🤷. This may work just fine in CI with the defined IP but switching to named lookup in Docker networking fixed the error I received above.

Reply:
Disregard. Something must have been amiss in my Docker setup because now it's working by IP:

```sh
➜ docker network inspect net-manylinux
[
    {
        "Name": "net-manylinux",
        "Id": "96d832aff6345828c99f19f2825342154774c8546a56f97d77b7c0d5fe22ff24",
        "Created": "2022-06-07T20:41:34.108436384Z",
        "Scope": "local",
        "Driver": "bridge",
        "EnableIPv6": false,
        "IPAM": {
            "Driver": "default",
            "Options": {},
            "Config": [
                {
                    "Subnet": "172.22.0.0/16",
                    "Gateway": "172.22.0.1"
                }
            ]
        },
        "Internal": false,
        "Attachable": false,
        "Ingress": false,
        "ConfigFrom": {
            "Network": ""
        },
        "ConfigOnly": false,
        "Containers": {
            "0a19cdd7a1c6c7c6847ef8306c774f70574326dfbb52b340701f02b50a7b4034": {
                "Name": "postgres",
                "EndpointID": "5789c4e441eef7902a2e1d78a3b4713d7b8da4deed26464dd756c635f2a29767",
                "MacAddress": "02:42:ac:16:00:02",
                "IPv4Address": "172.22.0.2/16",
                "IPv6Address": ""
            },
            "2cd92828c7a2edc692730fbeec3e805f67e2556d1865e55fd7bfee37376104a0": {
                "Name": "manylinux",
                "EndpointID": "8ec1c44c02f51854894183f616e1b01efcdab6f5433e2ea6fc6601279d039ca7",
                "MacAddress": "02:42:ac:16:00:03",
                "IPv4Address": "172.22.0.3/16",
                "IPv6Address": ""
            }
        },
        "Options": {},
        "Labels": {}
    }
]
```
```sh
[root@2cd92828c7a2 psycopg2]# PLAT=manylinux_2_28_aarch64 PACKAGE_NAME=psycopg2-binary PSYCOPG2_TESTDB=postgres PSYCOPG2_TESTDB_HOST=172.22.0.2 PSYCOPG2_TESTDB_USER=postgres PSYCOPG2_TESTDB_PASSWORD=password PSYCOPG2_TEST_FAST=1 ./scripts/build/build_manylinux_2_28.sh
+ PYTHON_VERSION=cp38
++ find /opt/python -maxdepth 1 -name 'cp38-*' -print -quit
+ pythonLocation=/opt/python/cp38-cp38
+ export PATH=/opt/python/cp38-cp38/bin:/usr/share/Modules/bin:/opt/rh/gcc-toolset-11/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ PATH=/opt/python/cp38-cp38/bin:/usr/share/Modules/bin:/opt/rh/gcc-toolset-11/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+++ dirname ./scripts/build/build_manylinux_2_28.sh
++ cd ./scripts/build
++ pwd
#...
++ dirname /usr/lib64/libpq.so.5
+ mv -v /usr/lib64/libpq.so.5 /libpqbak//usr/lib64
renamed '/usr/lib64/libpq.so.5' -> '/libpqbak//usr/lib64/libpq.so.5'
+ cd /app/psycopg2
+ for pyver in $PYVERS
+ pybin=/opt/python/cp310-cp310/bin
+ /opt/python/cp310-cp310/bin/pip install psycopg2-binary --no-index -f /app/psycopg2/dist/psycopg2-2.9.3
Looking in links: /app/psycopg2/dist/psycopg2-2.9.3
Processing ./dist/psycopg2-2.9.3/psycopg2_binary-2.9.3-cp310-cp310-manylinux_2_28_x86_64.whl
Installing collected packages: psycopg2-binary
Successfully installed psycopg2-binary-2.9.3
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
+ /opt/python/cp310-cp310/bin/python -c 'import psycopg2; print(psycopg2.__version__)'
2.9.3 (dt dec pq3 ext lo64)
+ /opt/python/cp310-cp310/bin/python -c 'import psycopg2; print(psycopg2.__libpq_version__)'
140003
+ /opt/python/cp310-cp310/bin/python -c 'import psycopg2; print(psycopg2.extensions.libpq_version())'
140003
+ [[ -n '' ]]
+ /opt/python/cp310-cp310/bin/python -c 'import tests; tests.unittest.main(defaultTest='\''tests.test_suite'\'')'
ss..................s.s.s..........ss.........ss.........s..ss.s...s.ssssssssssssssssss.................................ss................s.....s....s........s........s..s.s......s.s....................s.....s.......................................................................s.........................................................................................ss.......ss......................ss.s......................................s....sssss.sssssssssssss..................................................ssssssssss......................................................ss.....................................................s.........s.ssss.sssss...............s..............................................................
----------------------------------------------------------------------
Ran 738 tests in 4.883s

OK (skipped=96)
+ for pyver in $PYVERS
+ pybin=/opt/python/cp311-cp311/bin
#...
```